python/pyspark/streaming/dstream.py

```diff
@@ -602,13 +602,17 @@ class TransformedDStream(DStream):
     Multiple continuous transformations of DStream can be combined into
     one transformation.
     """
-    def __init__(self, prev, func):
+    def __init__(self, prev, func, transformFunc=None):
```

Comment:
I dont. @davies could you please take a look?
Also, could you elaborate on what were the deser errors you were seeing. Under what conditions? How to reproduce?